stock_move_line_auto_fill/views/stock_picking.xml

```diff
@@ -16,9 +15,18 @@
                     string="AutoFill"
                     attrs="{'invisible': ['|', ('action_pack_op_auto_fill_allowed','=', False), ('auto_fill_operation', '=', True)]}"
                     help="This button will automatically fill all operations that have no tracking set on the product, no processed qty and no selected package."
+                    groups="stock.group_stock_user"
+                />
+                <field
+                    name="action_pack_op_auto_fill_allowed"
+                    invisible="1"
+                    groups="stock.group_stock_user"
+                />
+                <field
+                    name="auto_fill_operation"
+                    invisible="1"
+                    groups="stock.group_stock_user"
```

Comment:
Not sure here and above but if field is invisible, define a specific group have no effect !?

Reply:
indeed, must be useless
@syera94 Can you remove these groups on the invisible fields ?